Formula/ddns-go.rb

```diff
@@ -0,0 +1,29 @@
+class DdnsGo < Formula
+  desc "Simple and easy-to-use DDNS"
+  homepage "https://github.com/jeessy2/ddns-go"
+  url "https://github.com/jeessy2/ddns-go/archive/refs/tags/v5.3.1.tar.gz"
+  sha256 "afca100b1b2d2b5bdcb0f6d061709136a567978e9cca85b9c0867d9219ead3f5"
+  license "MIT"
+  head "https://github.com/jeessy2/ddns-go.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=v#{version}
```

Comment:
Since we set the version by passing a flag, we should check the version output in the `test` to make sure we did it properly.

Reply:
Agreed. However, there is a missing flag for outputting the version. I have already opened jeessy2/ddns-go#722.